app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+
+from app.errors import (
+    NotWearingMaskError,
+    NotVaccinatedError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
+        if not visitor.get("vaccine", 0):

```

Comment:
The condition `if not visitor.get("vaccine", 0):` uses an unnecessary comparison to `0`. According to the checklist, you should avoid comparing a variable to `0` if not needed. Instead, you can simply use `if "vaccine" not in visitor:` to check if the 'vaccine' key is present in the visitor dictionary.